rest/config.go

```diff
@@ -1023,9 +1023,28 @@ func RunServer(config *ServerConfig) {
 	}
 
 	sc := NewServerContext(config)
-	for _, dbConfig := range config.Databases {
+	uuids := make(map[string][]string)
```

Comment:
Rename to make the purpose of the map clearer, and initialize enough starting capacity for each database.

```suggestion
	bucketUUIDToDbs := make(map[string][]string, len(config.Databases))
```

Reply:
Makes sense; this is fixed in latest revision.